commerce_coordinator/apps/commercetools/utils.py

```diff
@@ -144,3 +147,87 @@ def translate_stripe_refund_status_to_transaction_status(stripe_refund_status: s
         'failed': TransactionState.FAILURE,
     }
     return translations.get(stripe_refund_status.lower(), stripe_refund_status)
+
+
+def send_refund_notification(user, order_id):
+    """
+    Notify the support team of the refund request.
+
+    Returns:
+        bool: True if we are able to send the notification.  In this case that means we were able to create
+              a ZenDesk ticket
+    """
+
+    tags = ['auto_refund']
+
+    # Build the information for the ZenDesk ticket
+    student = user
+    subject = "[Refund] User-Requested Refund"
+    body = generate_refund_notification_body(student, order_id)
+    requester_name = student.username
```

Comment:
Since it is a CT customer object. We need to modify this to fetch username like:

```suggestion
    requester_name = student.custom.fields[EdXFieldNames.LMS_USER_NAME]
```

There is also a utility in CC for this `get_edx_lms_user_name()`